### ISSUE TITLE
How define a style's theme ?

### ISSUE THREAD

**Issue by mrpierrot** [nspitko/hlimgui#26]
Hi ! 
When using the "Style Editor" provide by Dear ImGui, I can get an export for theming my app : 

````
ImVec4* colors = ImGui::GetStyle().Colors;
colors[ImGuiCol_Text]                   = ImVec4(1.00f, 1.00f, 1.00f, 1.00f);
colors[ImGuiCol_TextDisabled]           = ImVec4(0.50f, 0.50f, 0.50f, 1.00f);
colors[ImGuiCol_WindowBg]               = ImVec4(0.06f, 0.06f, 0.06f, 0.94f);
colors[ImGuiCol_ChildBg]                = ImVec4(0.00f, 0.00f, 0.00f, 0.00f);
....
````

I would like to convert it in a valid haxe code. But I encounter some difficulties. An exemple

````
var colors = ImGui.getStyle().Colors;
//ImGui.styleColorsLight();
trace("x "+colors.Text.x);
trace("y "+colors.Text.y);
trace("z "+colors.Text.z);
trace("w "+colors.Text.w);
trace(ImGui.getStyleColorVec4(ImGuiCol.Text));
````
When I run it, I get : 

> Test.hx:228: x 3.000000000
Test.hx:229: y 1.000000000
Test.hx:230: z -135270416.000000000
Test.hx:231: w 1.250000000
Test.hx:232: { x: 1.000000000, y: 1.000000000, z: 1.000000000, w: 1.000000000 }

Here colors.Text seems a kind of ImVec4S but with strange value. Clearly it's not a color. 

How can I make a thing like that : 

````
colors.Text = new ImVec4(0.78, 0.77, 0.77, 1.00); 
colors.TextDisabled = new ImVec4(0.78, 0.77, 0.77, 1.00); 
...
````

Thank you.

**Comment by nspitko** [comment]
Some new style fields were added at some point and we missed them, causing our struct to fall out of alignment. Fixed under 979426f1111d19e86b01bfbb0646b2ba2bb89073

**Comment by mrpierrot** [comment]
Thank you very much. I confirme that works fine for me.
